### DISCUSSION TITLE
Is useCSSLayers: true required?

### DISCUSSION THREAD

**Discussion by steida** [facebook/stylex#316] [Q&A] [answered]
I'm curious why it has to be enabled. Thank you.

**Comment by necolas** [comment, marked as answer]
Because layers isn't supported in older browsers so we use a polyfill by default

**Comment by nmn** [comment]
Additionally, the way that CSS layers work, any "unlayered" styles will override layered styles. This means that a lot of people are confused when their "reset" styles override StyleX styles.

Using native CSS layer may *look* cleaner, but the polyfill actually causes fewer surprises.
